<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #dcdcdc;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>887</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="pushButton_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>T</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QPushButton" name="pushButton_20">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="pushButton_18">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="pushButton_19">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>O</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pushButton_17">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>U</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>Q</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid white; border-radius: 20px;
	font-size: 20px; font-family: Arial;
	background-color: #f0eeee;
}

QPushButton:pressed {
	border: 2px solid red; border-radius: 30px;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	border: 1px solid black; border-radius: 10px;
}

QPushButton : hover {
	cursor: pointer;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	border: 1px solid black; border-radius: 10px;
}

QPushButton : hover {
	cursor: pointer;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
